### PR TITLE
Move Stanford's CMS utilities tests

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
CC: @dcadams @caesar2164 @caseylitton @kluo

**Comment by stvstnfrd** [comment]
I'd like one or two confirmations on this please :)

File: openedx/stanford/cms/djangoapps/contentstore/tests/test_utilities.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def compare_utilities(self, persisted, request):
 
     def test_get_utilities(self):
         """ Tests the get utilities method and URL expansion. """
+        self.assertTrue(False)
```
Comment (caesar2164) [inline openedx/stanford/cms/djangoapps/contentstore/tests/test_utilities.py]: maybe remove testing assert? ;)

Comment (caesar2164) [inline openedx/stanford/cms/djangoapps/contentstore/tests/test_utilities.py]: (once you see that it's working and you've squashed)

Comment (stvstnfrd) [inline openedx/stanford/cms/djangoapps/contentstore/tests/test_utilities.py]: @caesar2164 Yeah, that's why it's in a separate commit that explicitly mentions it fails on purpose ;)

**Comment by stvstnfrd** [comment]
@caesar2164 Updated.

**Comment by caesar2164** [comment]
ok cool, 👍

**Pull request merged**